winpr/libwinpr/synch/wait.c

```diff
@@ -109,7 +109,7 @@ static long long ts_difftime(const struct timespec *o,
 #if !defined(HAVE_PTHREAD_GNU_EXT)
 #include <pthread.h>
 
-#if defined(__FreeBSD__) || defined(sun)
+#if defined(__FreeBSD__) || defined(sun) || defined(__OpenBSD__)
```

Comment:
Hi,

It's declared as non-static in /usr/include/pthread.h on OpenBSD 5.6


Thanks,
Bryan

On Tue, Apr 21, 2015 at 3:11 PM, akallabeth <notifications@github.com>
wrote:

> In winpr/libwinpr/synch/wait.c
> <https://github.com/FreeRDP/FreeRDP/pull/2562#discussion_r28811878>:
>
> > @@ -109,7 +109,7 @@ static long long ts_difftime(const struct timespec *o,
> >  #if !defined(HAVE_PTHREAD_GNU_EXT)
> >  #include <pthread.h>
> >
> > -#if defined(__FreeBSD__) || defined(sun)
> > +#if defined(__FreeBSD__) || defined(sun) || defined(__OpenBSD__)
>
> @bceverly <https://github.com/bceverly> Maybe you have some insight, what
> is causing the issue with this function being declared static?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/FreeRDP/FreeRDP/pull/2562/files#r28811878>.
>